src/script/components/copyToClipboard.ts

```diff
@@ -18,14 +18,17 @@
  */
 
 class CopyToClipboard {
-  constructor(params) {
+  // tslint:disable-next-line
```

Comment:
Why disable TSLint here?

Reply:
`'text' is declared but its value is never read.`

It's used in the template below, but TypeScript can't see that.. if you have anything better, we are open :)

Reply:
Travis is still complaining about it..

Reply:
Make it public :)

Reply:
It was that easy! Thanks